js/popups/tablemenupopup.ts

```diff
@@ -359,6 +362,7 @@ export class TableMenuPopup {
         this.skipDeals(!this.skipDeals());
         const currentTable = this.currentTableProvider.currentTable();
         currentTable.toggleSkipDeals(this.skipDeals());
+        app.closePopup();
```

Comment:
use `this.confirm()`. That should hide access to global app. and better indicate that we sucessfuly close popup. It is potentially left of er from earlier code, when we don't have shared popups code. Maybe we could also call `this.close()` instead of `app.closePopup()` in the `confirm()` method